plonetheme/barceloneta/theme/index.html

```diff
@@ -13,22 +13,21 @@
   <link rel="apple-touch-icon-precomposed" href="barceloneta-apple-touch-icon-precomposed.png" />
 </head>
 
-<body>
+<body class="d-flex flex-column h-100">
   <section id="portal-toolbar">
   </section>
 
-  <div class="outer-wrapper">
-    <header id="content-header">
+  <div class="outer-wrapper flex-shrink-0">
+    <header id="content-header" class="container d-flex pt-3 flex-column justify-content-between align-items-md-start flex-md-row">
 
-      <div id="portal-top" class="container">
+      <div id="portal-top" class="d-flex">
       </div>
 
-      <div id="portal-logo-wrapper" class="container d-flex">
-        <div id="portal-logo"></div>
+      <div id="portal-logo-wrapper" class="pb-3 d-flex justify-content-between align-items-center order-md-first w-auto">
```

Comment:
why does the logo wrapper need `justify-content-between` and `align-items-center` ?

Reply:
some reponsive issue... or can one be removed?

Reply:
yes right ... wrapper classes are needed for mobile view. 👍🏼